### PR TITLE
Show space default image image and hide member count in spaces management

### PR DESCRIPTION
**Pull request by JammingBen**
## Related Issue

- https://github.com/owncloud/web/issues/8222

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/31573/12/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/31573/tracing/participant-management-alice-2023-1-13-03-36-41.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/31573/tracing/participant-management-anonymous-2023-1-13-03-37-14.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/31573/tracing/participant-management-brian-2023-1-13-03-36-51.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/31573/tracing/participant-management-carol-2023-1-13-03-37-03.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/31573/tracing/participant-management-marie-2023-1-13-03-36-55.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/31573/tracing/participant-management-richard-2023-1-13-03-36-58.zip 


</p></details>
:boom: The e2e-ocis tests pipeline failed. The build has been cancelled.

**Review by kulmann — CHANGES_REQUESTED** [review]
I'd like to see the big space icon (which we also have if no image exists). Otherwise the sidebar makes a super empty impression.

File: packages/web-pkg/src/components/sideBar/Spaces/Details/SpaceDetails.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
   <div id="oc-space-details-sidebar">
     <div class="oc-space-details-sidebar-image oc-text-center">
-      <oc-spinner v-if="loadImageTask.isRunning" />
-      <div v-else-if="spaceImage" class="oc-position-relative">
+      <oc-spinner v-if="showSpaceImage && loadImageTask.isRunning" />
```
Comment (kulmann) [inline packages/web-pkg/src/components/sideBar/Spaces/Details/SpaceDetails.vue:4]: Wouldn't it make more sense to have an early return in the `loadImageTask` if `showSpaceImage === false`?

Comment (kulmann) [reply packages/web-pkg/src/components/sideBar/Spaces/Details/SpaceDetails.vue:4]: Then the template can remain unchanged (this line and next line).

Comment (JammingBen) [reply packages/web-pkg/src/components/sideBar/Spaces/Details/SpaceDetails.vue:4]: Alright, since I need to rebase anyway 😄

**Pull request merged**